### PR TITLE
Re-edit of PGD group 15

### PR DESCRIPTION
**Pull request by ebgitelman**

### CODE REVIEW

File: product_docs/docs/pgd/5/backup.mdx [hunk 1]
```diff
@@ -120,9 +120,9 @@ applied to `N2` until `T2`.
...
 from a single master in COMMIT order. Thus, PITR is possible by 
 scanning through changes until one particular point in time (PIT) is reached.
-With this scheme, you can restore one node to a single point in time
+With this scheme, you can restore one node to a single PIT
 from its viewpoint, for example, `T1`. However, that state doesn't include other
-data from other nodes that  committed near that time but had not yet
+data from other nodes that committed near that time but had not yet
 arrived on the node. As a result, the recovery might be considered to
 be partially inconsistent, or at least consistent for only one
 replication origin.
```
Comment (ebgitelman) [inline product_docs/docs/pgd/5/backup.mdx:128]: In the next sentence, it wasn't clear to me what "this" is. Is it PITR? 

To request PITR, use the standard syntax: 

?

File: product_docs/docs/pgd/5/parallelapply.mdx [hunk 2]
```diff
@@ -70,11 +70,11 @@ performance.
```
Comment (ebgitelman) [inline product_docs/docs/pgd/5/parallelapply.mdx:70]: Above, line 29, this sentence: You can only change
[`bdr.max_writers_per_subscription`](/pgd/latest/reference/pgd-settings#bdrmax_writers_per_subscription)
with a server restart.

Can this be rewritten in the positive? Would this be accurate? 

Changing [`bdr.max_writers_per_subscription`](/pgd/latest/reference/pgd-settings#bdrmax_writers_per_subscription) requires a server restart.

File: product_docs/docs/pgd/5/parallelapply.mdx [hunk 3]
```diff
@@ -70,11 +70,11 @@ performance.
 
 The `nprovisional_waits` value reflects the number of operations on the same
 tuples being performed by concurrent apply transactions. These are provisional
-waits that aren't actually waiting yet but could. 
+waits that aren't actually waiting yet but could.
```
Comment (ebgitelman) [inline product_docs/docs/pgd/5/parallelapply.mdx:73]: Can we clarify the conditional here? Like "...but could be if..."

File: product_docs/docs/pgd/5/parallelapply.mdx (hunk 2 above)
Comment (djw-m) [reply product_docs/docs/pgd/5/parallelapply.mdx:70]: Changing [bdr.max_writers_per_subscription](https://github.com/pgd/latest/reference/pgd-settings#bdrmax_writers_per_subscription) requires a server restart to take effect.

File: product_docs/docs/pgd/5/parallelapply.mdx (hunk 3 above)
Comment (djw-m) [reply product_docs/docs/pgd/5/parallelapply.mdx:73]:
```suggestion
waits that aren't actually waiting yet but could start waiting.
```

File: product_docs/docs/pgd/5/backup.mdx (hunk 1 above)
Comment (djw-m) [reply product_docs/docs/pgd/5/backup.mdx:128]: Do you mean this sentence?

With this scheme, you can restore one node to a single PIT
from its viewpoint, for example, `T1`.

Comment (djw-m) [reply product_docs/docs/pgd/5/backup.mdx:128]: I think... 

"With PostgreSQL POTR, you can..."

File: product_docs/docs/pgd/5/backup.mdx [hunk 4]
```diff
@@ -270,7 +270,7 @@ If a physical backup was created with `pg_basebackup`, replication slots
 are omitted from the backup.
 
 Some other backup methods might preserve replications slots, likely in
-outdated or invalid states. Once you restore the backup, use this to drop all replication slots:
+outdated or invalid states. Once you restore the backup, use these commands to drop all replication slots:
```
[the remark this answers is not in the document]
Comment (djw-m) [reply product_docs/docs/pgd/5/backup.mdx:273]: !!!Warning
Never use these commands to drop replication slots on a live PGD node

**Comment by ebgitelman** [comment]
@djw-m Incorporated your comments and ready to approve. Thanks for your changes!

**Review by djw-m — APPROVED** [review]
LGTM

**Pull request merged**